packages/pds/src/auth.ts

```diff
@@ -324,3 +361,13 @@ export enum AuthStatus {
   Invalid,
   Missing,
 }
+
+const createPrivateKeyObject = async (
```

Comment:
mer it's a shame we have to convert between these :/

If we need to use their key format, I don't think it'd be wrong to just set the config value as a pem/der formatted key 🤔

Reply:
Going to comment below in context of the multicodecs idea 👍